src/app/pages/forgot-password/update-password-form/update-password-form.component.ts

```diff
@@ -23,34 +24,69 @@ export class UpdatePasswordFormComponent implements OnInit {
    */
   @Output() submitPassword = new EventEmitter<{ password: string }>();
 
-  password: string;
-
-  form: FormGroup;
+  updatePasswordForm = new FormGroup({});
+  fields: FormlyFieldConfig[];
   submitted = false;
 
+  constructor(private config: FormlyConfig) {}
+
   ngOnInit() {
-    this.form = new FormGroup(
+    this.fields = [
+      {
+        key: 'password',
+        type: 'ish-password-field',
+        wrappers: [...(this.config.getType('ish-password-field').wrappers ?? []), 'description'],
```

Comment:
with #620 you can use the templateOptions.postWrappers attribute to append a wrapper to a field config. This way you don't have to inject the config service anymore.